operations/rates.md

```diff
@@ -124,6 +124,7 @@ Extent of data to be returned.
 | `GroupId` | string | required | Unique identifier of `Rate Group` where the rate belongs. |
 | `ServiceId` | string | required | Unique identifier of the `Service`. |
 | `BaseRateId` | string | optional | Unique identifier of the base `Rate`. |
+| `IsBaseRate` | boolean | required | Whether the rate is a base rate. |
```

Comment:
It's a minor point, and I know we've used "Whether" in the past plenty of times, but I prefer this wording for booleans:
```suggestion
| `IsBaseRate` | boolean | required | Indicates if this is a base rate. |
```